llvm/lib/Target/RISCV/RISCVInstrFormats.td

```diff
@@ -49,6 +49,7 @@ def InstFormatCB     : InstFormat<15>;
 def InstFormatCJ     : InstFormat<16>;
 def InstFormatOther  : InstFormat<17>;
 def InstFormatCVHwlp : InstFormat<18>;
+def InstFormatCVMac  : InstFormat<19>;
```

Comment:
Is it the case that the instructions in this extension all have one custom format?

Reply:
They use different custom op codes and the mac+msu instructions have other encoding structure.

IIUC those `InstFormat`s are only used for relocations? Would it be better to use `InstFormatOther` here?

Reply:
These map to the instruction type/immediate types that the RISC-V spec specifies, 'R-type', 'I-type', etc. and should be enough to distinguish the entire instruction format.

If CORE-V is defining its own instruction formats, then we should use those type names, otherwise Other may be more suitable as a starting point

Reply:
I see. I don't think CORE-V defines an instruction format, but I could be mistaken. I wuld use `InstrFormatOther` for mac, but keep using `InstFormatHwlp` for simpler relocations code.